google-cloud-clients/google-cloud-core/src/test/java/com/google/cloud/IdentityTest.java

```diff
@@ -141,16 +141,16 @@ public void testValueOfEmpty() {
     Identity.valueOf("");
   }
 
-  @Test(expected = IllegalArgumentException.class)
-  public void testValueOfThreePart() {
-    Identity.valueOf("a:b:c");
-  }
-
   @Test
   public void testUnrecognizedToString() {
     assertEquals("a:b", Identity.valueOf("a:b").strValue());
   }
 
+  @Test
+  public void testValueOfThreePart() {
+    assertEquals("a:b:c", Identity.valueOf("a:b:c").strValue());
```

Comment:
I meant to make sure that the the type and value are split the way we expect - type is `a` and value is `b:c`